homeassistant/components/fibaro/scene.py

```diff
@@ -33,6 +33,13 @@ async def async_setup_entry(
 class FibaroScene(FibaroDevice, Scene):
     """Representation of a Fibaro scene entity."""
 
+    def __init__(self, fibaro_device: Any) -> None:
+        """Initialize the Fibaro scene."""
+        super().__init__(fibaro_device)
+
+        # All scenes are shown on hub device
+        self._attr_device_info = self.controller.hub_device_info
```

Comment:
You don't need to give the full device info each time. You could just have `identifiers={(DOMAIN, info.serialNumber)}` and it would connect it to the hub.

Reply:
Thanks for the hint, I was not aware of that possibility. I changed the code accordingly. As now the device info of the hub is only needed once I create it now on the fly.